.github/workflows/scripts/tag_check.sh

```diff
@@ -0,0 +1,45 @@
+#!/bin/bash
+
+set -euo pipefail
+
+MAIN_OUTPUT_TAG=$1
+MAIN_OUTPUT_NEWTAG=$2
+MAIN_OUTPUT_PART=$3
+PRE_OUTPUT_TAG=$4
+PRE_OUTPUT_NEWTAG=$5
+PRE_OUTPUT_PART=$6
+
+echo "Outputs from running the action:" >>"$GITHUB_STEP_SUMMARY"
+echo "MAIN Tag: $MAIN_OUTPUT_TAG" >>"$GITHUB_STEP_SUMMARY"
+echo "MAIN New tag: $MAIN_OUTPUT_NEWTAG" >>"$GITHUB_STEP_SUMMARY"
+echo "MAIN Part: $MAIN_OUTPUT_PART" >>"$GITHUB_STEP_SUMMARY"
+echo "PRE Tag: $PRE_OUTPUT_TAG" >>"$GITHUB_STEP_SUMMARY"
+echo "PRE New tag: $PRE_OUTPUT_NEWTAG" >>"$GITHUB_STEP_SUMMARY"
+echo "PRE Part: $PRE_OUTPUT_PART" >>"$GITHUB_STEP_SUMMARY"
```

Comment:
A little trick with this to make it render in a nice table can be something like this:

```bash
OUTPUT_SUMMARY="\n
| Output       | Value                 | \n
| ------------ | --------------------- | \n
|              |                       | \n
| MAIN Tag     | ${MAIN_OUTPUT_TAG}    | \n
| MAIN New tag | ${MAIN_OUTPUT_NEWTAG} | \n
| MAIN Part    | ${MAIN_OUTPUT_PART}   | \n
| PRE Tag      | ${PRE_OUTPUT_TAG}     | \n
| PRE New tag  | ${PRE_OUTPUT_NEWTAG}  | \n
| PRE Part     | ${PRE_OUTPUT_PART}    | \n
"

echo -e "$OUTPUT_SUMMARY" >>"$GITHUB_STEP_SUMMARY"